server/src/routes/api/v1/auth/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Router, Request, Response, NextFunction } from "express";
```

Comment:
What do you think about renaming index.ts? I'm not sure why we use index if these files contain routes? With recipes I called this recipe.routes.ts